features/support/steps_helper.rb

```diff
@@ -50,3 +50,16 @@
 Then('I choose option {string}') do |field|
   choose(field.parameterize(separator: '_'), allow_label_click: true)
 end
+
+# This can be used to display the state of the application
+# Usage:
+# Then I display the state of the application from 123
+#
+Then('I display the state of the application from {int}') do |label|
```

Comment:
More for my own understanding than anything else, what does 123 represent here?

Reply:
Oh, it's just a label.  I found I was having to debug a cuke by putting one in at every page save, so it was useful to have a label on it.

Reply:
I should probably have made it a string label with a better usage message